argocd/overlays/moc-infra/projects/apicurio-registry.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
```

Comment:
Can you add a label:
```yaml
  labels:
    project-template: global
```

This should have been reflected in the docs (but isn't), I'll update docs accordingly.

Reply:
done, thank you